Makefile

```diff
@@ -45,7 +45,7 @@ help: ## Display this help.
 ##@ Development
 
 manifests: ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=externaldns-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
We decided to go for `external-dns-operator` right?

Reply:
fixed. Thanks!